src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1210,4 +1213,20 @@ public static synchronized void updateBurns()
         modpacks.add("vanilla");
         return modpacks;
     }
+
+    public static IPacket<?> cleanPacketForPossibleVanillaConnection(ServerPlayNetHandler netHandler, IPacket<?> packet)
+    {
+        if (NetworkHooks.getConnectionType(() -> netHandler.netManager) == ConnectionType.VANILLA)
```

Comment:
Would this be cleaner using [`NetworkHooks.isVanillaConnection(netHandler.netManager)`](https://github.com/MinecraftForge/MinecraftForge/blob/1.16.x/src/main/java/net/minecraftforge/fml/network/NetworkHooks.java#L122)

Reply:
You would be correct. Will change.